src/pages/standard-report/HtmlReportAssets.js

```diff
@@ -27,3 +27,23 @@ export const SCRIPT_FILES = [
     '/dhis-web-commons/javascripts/date.js',
     '/dhis-web-commons/oust/oust.js',
 ]
+
+export const PAGE_STYLES = `
+    html {
+        line-height: 1.15;
+        text-size-adjust: 100%;
+    }
+    body {
+        font-family: Roboto, sans-serif;
+        box-sizing: inherit;
+    }
```

Comment:
Are these additional styles that you added in this PR? They're not in the css code that's been removed in the other file